test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/ApiControllerActionDiscoveryTest.cs

```diff
@@ -57,7 +57,7 @@ public void GetActions_ApiControllerWithoutControllerSuffix_IsNotController()
             var controllerType = typeof(TestControllers.Blog).GetTypeInfo();
             var actions = results.Where(ad => ad.ControllerTypeInfo == controllerType).ToArray();
 
-            Assert.Empty(actions);
+            Assert.Equal(2,actions.Count());
```

Comment:
Fix spacing and don't use Linq for a simple array. Probably best to remove the unnecessary `ToArray()` just above.